src/modules/likes.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-plusplus */
```

Comment:
please it would be better to remove the diable linter cause it ensures that you write a clean code